https://bugzilla.mozilla.org/show_bug.cgi?id=1771273
https://hg.mozilla.org/projects/nss/raw-rev/21e7aaa1f7d94bca15d997e5b4c2329b32fad21a

# HG changeset patch
# User Sergei Trofimovich <slyich@gmail.com>
# Date 1653552519 0
# Node ID 21e7aaa1f7d94bca15d997e5b4c2329b32fad21a
# Parent  ad1046e9eee5f5dc17dac7c9343e2f7f0da44b4e
Bug 1771273 - cpputil/databuffer.h: add missing <cstdint> include r=nss-reviewers,mt

Without the change build fails on this week's gcc-13 snapshot as:

    ../../cpputil/databuffer.h:20:20: error: 'uint8_t' does not name a type
       20 |   DataBuffer(const uint8_t* d, size_t l) : data_(nullptr), len_(0) {
          |                    ^~~~~~~
    ../../cpputil/databuffer.h:14:1: note: 'uint8_t' is defined in header '<cstdint>'; did you forget to '#include <cstdint>'?
       13 | #include <iostream>
      +++ |+#include <cstdint>
       14 |

Differential Revision: https://phabricator.services.mozilla.com/D147404

diff --git a/cpputil/databuffer.h b/cpputil/databuffer.h
--- nss/cpputil/databuffer.h
+++ nss/cpputil/databuffer.h
@@ -6,16 +6,17 @@
 
 #ifndef databuffer_h__
 #define databuffer_h__
 
 #include <algorithm>
 #include <cstring>
 #include <iomanip>
 #include <iostream>
+#include <cstdint>
 
 namespace nss_test {
 
 class DataBuffer {
  public:
   DataBuffer() : data_(nullptr), len_(0) {}
   DataBuffer(const uint8_t* d, size_t l) : data_(nullptr), len_(0) {
     Assign(d, l);

